receiver/dummyreceiver/README.md

```diff
@@ -0,0 +1,23 @@
+# Dummy Receiver
+
+| Status      |                            |
+|-------------|----------------------------|
+| stability   | alpha: metrics             |
+| Code Owners | kyma-project/observability |
+
+Dummy Receiver is an OTel Collector receiver that generates dummy telemetry data. At the moment it only supports metrics. It is useful for testing
+the OTel Collector pipeline.
+
+## How It Works
+
+It generates dummy metrics and sends them to the OTel Collector pipeline.
+
+## Configuration
+
+Below is an example of the configuration:
+
+```yaml
+receivers:
+  dummy:
+    interval: 2s
+```
```

Comment:
Maybe also list the resulting metric names with the metric type, and what resource attributes and metric attributes they will have. For someone new, it will be nice to directly see what to expect. And indicate if there are dynamic values or if it is all constant.

Reply:
@hisarbalik Can we use the same mechanism that you used for `kymastatsreceiver` here?

Reply:
i added metrics information to the metadata.yaml and linked it in the readme